interface/wx/app.h

```diff
@@ -965,6 +965,64 @@ class wxApp : public wxAppConsole
     */
     bool ProcessMessage(WXMSG* msg);
 
+
+    /**
+        Possible parameters for SetAppearance().
+
+        @since 3.3.0
+    */
+    enum class Appearance
+    {
+        System, ///< Use system default appearance.
+        Light,  ///< Use light appearance.
+        Dark    ///< Use dark appearance.
+    };
+
+    /**
+        Possible values returned by SetAppearance().
+
+        @since 3.3.0
+    */
+    enum class AppearanceResult
+    {
+        Failure,     ///< Changing the appearance failed.
+        Success,     ///< Appearance was successfully changed.
+        CannotChange ///< Appearance can't be changed any more.
+    };
+
+    /**
+        Request using either system default or explicitly light or dark theme
+        for the application.
+
+        Under GTK and macOS applications use the system default appearance by
+        default, and so it is only useful to call this function with either
+        Appearance::Light or Appearance::Dark parameters if you need to
+        override the default system appearance. The effect of calling this
```

Comment:
I wonder if we should note the existence of `wxSystemAppearance` here and analogically, reference this method in `wxSystemAppearance` docs.
```suggestion
        override the default system appearance (see wxSystemAppearance). The effect of calling this
```

The wxWidgets API is very extensive and sometimes it may be hard to find the correct method/class that does what is wanted...

Reply:
Thanks, I've added xrefs in both directions now.